### DISCUSSION TITLE
1:9 Balun

### DISCUSSION THREAD

**Discussion by OE9SAU** [DG1JAN/UniBalun#4] [Q&A]
Hallo Jan, 
wie wickelst Du hier einen 1:9 Balun ?
Grüsse Martin

**Comment by dragonito** [comment]
Moin, eigentlich ist es nen 1:9 UnUn, ich habe da nen [Schaubild ](http://3.bp.blogspot.com/_Yu1CtHcSjUs/TNqP5fzQuuI/AAAAAAAAABg/_J7NrfVpW7M/s1600/wickeln.gif) in Google gefunden, vielleicht hilft es Dir weiter.

**Reply by OE9SAU** [reply]
Hallo Danke für de Rückmeldung.

**Reply by DG1JAN** [reply]
Hallo und sorry für die späte Rückmeldung. War leider ein paar Tage auf Dienstreise.

Wie von @dragonito schon gepostet (vielen Dank dafür), kannst Du den 1:9 trifilar wickeln, oder analog zum 1:49 mit einer Windung von 3-zu-9. Siehe z.B. auch hier: https://www.qsl.net/dk7zb/Baluns/MTFT.htm

73 de Jan

**Comment by OE9SAU** [comment]
Hallo Jan, kein Thema. Danke für den Link.
73 de martin
